src/features/movies/moviesSaga.js

```diff
@@ -6,13 +6,16 @@ import {
     handleFetchingError,
     finishFetching,
     fetchMovies,
+    setMoviesGenres,
 } from "./moviesSlice";
 
 function* fetchMoviesHandler() {
     yield put(startFetching());
     try {
         const moviesList = yield axios.get("https://api.themoviedb.org/3/movie/popular?api_key=b6338a2fff00b848e44db36dd695b802&language=en-US&page=1");
+        const movieGenres = yield axios.get("https://api.themoviedb.org/3/genre/movie/list?api_key=b6338a2fff00b848e44db36dd695b802&language=en-US");
```

Comment:
It would be better if we named this variable moviesGenres for coherency.

Reply:
okey right